v2.0/getting-started/kubernetes/index.md

```diff
@@ -1,31 +1,72 @@
 ---
-Calico can be used as a network plugin for Kubernetes to provide connectivity and network policy in a Kubernetes cluster.
-Calico allows you to enforce fine-grained network security policy natively in Kubernetes. The video below shows a quick demonstration of Calico policy in action.
 
-[![IMAGE ALT TEXT](http://img.youtube.com/vi/OE1n5PWtvMM/0.jpg)](http://www.youtube.com/watch?v=OE1n5PWtvMM "Calico network policy on Kubernetes")
+Calico drives pod connectivity and implements network policy for Kubernetes clusters.
+Calico allows you to enforce fine-grained network security policy natively in Kubernetes.
 
-## Getting Started
+Use the navigation bar on the left to view information on Calico for Kubernetes,
+or continue reading for recommended guides to get started.
 
-The following guides help you get started with Calico.
+## Requirements
 
-Integration guide
-------------------
+- The kube-proxy must be started in `iptables` proxy mode.  This is the default as of Kubernetes v1.2.0.
 
-- [Integration Guide]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/): Discusses adding Calico to an existing cluster, or integrating Calico with your own install scripts.
+## Installation Guides
 
-Quick-start guides
-------------------
-These guides let you get a cluster set up quickly, and walk you through using Calico for networking and network policy.
+There are several ways to install Calico in Kubernetes.
 
-- [CoreOS Vagrant]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/vagrant/)
-- [CoreOS on GCE]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/gce)
-- [CoreOS on AWS]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/aws)
+#### [Hosted Install Guide]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/hosted)
 
-## Requirements
+Calico's Hosted Install uses Kubernetes Manifests to install Calico for Kubernetes, using Kubernetes.
+Calico manifests ensure the necessary artifacts and configurations are added to each host, and the necessary services are started.
 
-- The kube-proxy must be started in `iptables` proxy mode.  This is the default as of Kubernetes v1.2.0.
+>Note: Hosted installs are new for Kubernetes v1.4 and Calico's Manifests are still in Beta. It will become
+the supported installation method going forward.
+
+#### [Manual Install Guide]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/)
+
+This guide details how to add manually add Calico to an existing, stock Kubernetes cluster.
+
+
+## Quickstart with "Calico-Ready" Clusters
+
+Calico maintains sample cloud-config files which can be used to launch a Kubernetes
+cluster ready for Calico to be installed on.
+
+The following guides walk through launching a CoreOS cluster using these cloud-config scripts on various cloud providers and locally.
+These Kubernetes clusters will not install Calico for users, leaving you to
```

Comment:
The guides actually do include a bit about installing Calico.

I'd reword this to say something like "The guides install Kubernetes, and then use the Calico hosted installation on top" or something similar but better.